src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -92,34 +92,31 @@ public ControlledResource validateControlledResourceAndAction(
     return controlledResource;
   }
 
-  public ControlledResource syncCreateControlledResource(
-      ControlledResource resource,
+  /** Starts a create controlled bucket resource, blocking until its job is finished. */
+  public ControlledGcsBucketResource syncCreateBucket(
+      ControlledGcsBucketResource resource,
       ApiGcpGcsBucketCreationParameters creationParameters,
       List<ControlledResourceIamRole> privateResourceIamRoles,
       AuthenticatedUserRequest userRequest) {
-    ApiJobControl syncJobControl = new ApiJobControl().id(UUID.randomUUID().toString());
-    String jobId =
-        createControlledResource(
-            resource,
-            creationParameters,
-            privateResourceIamRoles,
-            syncJobControl,
-            null,
-            userRequest);
+    JobBuilder jobBuilder =
+        commonCreationJobBuilder(
+                resource, privateResourceIamRoles, UUID.randomUUID().toString(), null, userRequest)
+            .addParameter(ControlledResourceKeys.CREATION_PARAMETERS, creationParameters);
+    String jobId = jobBuilder.submit();
```

Comment:
this can just be `jobBuilder.submitAndWait()` without all the code below

Reply:
thanks, done.